docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   web: &web_base
     build:
       context: .
-      dockerfile: Dockerfile.dev
```

Comment:
The main dockerfile runs in the production environment, compiles assets, etc. We should keep the dev file.

Reply:
Do you make use of the Dockerfile.dev? If not, my impression try to approximate our Dockerfile builds close to staging/production

Reply:
Hah, only recently when having to change it. If you're doing development in Docker, I think it makes sense to. That's what compose is used for, so I think we should keep it. Also, more gems are installed so more dependencies are needed. AFAIK no one develops using it, but they should be kept together. I have some local changes/fixes to push up for it in any case.